pom.xml

```diff
@@ -76,6 +76,13 @@
             <artifactId>branch-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- see https://github.com/jenkinsci/custom-folder-icon-plugin/issues/280 -->
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>config-file-provider</artifactId>
+            <version>959.vcff671a_4518b_</version>
```

Comment:
I believe that the version of the config file provider plugin is already provided by the plugin bill of materials.  
```suggestion
```